<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorTextoMainWindows</class>
 <widget class="QMainWindow" name="EditorTextoMainWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor Texto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txt_texto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mbr_principal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_archivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="mni_abrir"/>
    <addaction name="mni_guardar"/>
    <addaction name="separator"/>
    <addaction name="mni_salir"/>
   </widget>
   <addaction name="mnu_archivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mni_abrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="mni_guardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="mni_salir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
